src/display.h

```diff
@@ -270,6 +270,45 @@ void displayShottimer(void) {
     #endif
 }
 
+#if (ROTARY_MENU == 1) 
+
+void displayNumericalMenuSettingWithUnit(double temp, const char* name, const char* unit) {
+    u8g2.clearBuffer();
+    u8g2.setCursor(0, 0);
+    u8g2.print(name);
+    u8g2.print(": ");
+    u8g2.setCursor(0, 10);
+    u8g2.print(temp, 1);
+    u8g2.print(unit);
+    u8g2.setCursor(0, 25);
+    u8g2.println(langstring_pressToSave[0]);
+    u8g2.setCursor(0, 35);
+    u8g2.println(langstring_pressToSave[1]);
+    u8g2.sendBuffer();
+}
+
+void displayToggleMessage(const char* title, uint8_t mode) {
+    u8g2.clearBuffer();
+    u8g2.setCursor(0, 0);
+    u8g2.print(title);
+    
+    switch (mode) {
```

Comment:
For 1 and 0 this should probably not be a switch statement. 

if (mode == 1) will be a lot more readable. 

If there are more than 2 modes then a switch statement might be OK, but the magic numbers should then be replaced by an enum.